tests/utils/test_misc.py

```diff
@@ -0,0 +1,35 @@
+from kornia.core import tensor
+from kornia.utils.misc import (
+    differentiable_clipping,
+    differentiable_polynomial_floor,
+    differentiable_polynomial_rounding,
+)
+
+from testing.base import BaseTester
+
+
+class TestDifferentiableClipping(BaseTester):
+    def test_differentiable_clipping(self, device):
+        x = tensor([1.0, 6.0, 10.0, 12.0], device=device)
+        y = differentiable_clipping(x, min_val=5.0, max_val=10.0)
+        y_expected = tensor([4.9804, 6.0, 10.0, 10.0173], device=device)
+
+        self.assert_close(y, y_expected)
+
+
```

Comment:
can you add grad tests for each of the functions? just to ensure we are testing the differentiability here

it will look something like it:
```suggestion
def test_gradcheck(self, device=device):
    x = tensor([1.0, 6.0, 10.0, 12.0], device=device)
    self.gradtest(differentiable_clipping, (x, 5.0, 10.0))

```

Reply:
@johnnv1 Added `gradcheck`. One think I noticed that when `min_val` or `max_val` eqauls `x` then gradcheck for `differentiable_clipping` will fail. For Example the above one fails when `x=10.0` because `min_val=10.0`